source/Octostache.Tests/CalculationFixture.cs

```diff
@@ -33,16 +33,18 @@ public static IEnumerable<object[]> ConditionalIsSupportedData()
             yield return new object[] { "A+2", "7" };
             yield return new object[] { "A+B", "12" };
             yield return new object[] { "C+2", "#{C+2}" };
-            // In the TemplateParser.cs for Identifier and IdentifierWithoutWhitespace we include / and - as part of the identifier.
-            // There isn't a clean fix for this, but exlcuding these in the context of a calc operation would resolve this issue (and introduce problems for variables using - and / in calc operations).
-            //yield return new object[] { "B-2", "5" };
+            yield return new object[] { "{B}-2", "5" };
             yield return new object[] { "2-B", "-5" };
             yield return new object[] { "(B*2)-2", "12" };
             yield return new object[] { "2-(B*2)", "-12" };
-            yield return new object[] { "B/2", (7d / 2).ToString(CultureInfo.CurrentCulture) };
+            yield return new object[] { "{B}/2", (7d / 2).ToString(CultureInfo.CurrentCulture) };
             yield return new object[] { "2/B", (2d / 7).ToString(CultureInfo.CurrentCulture) };
```

Comment:
maybe consider simple tests like "2/4" and " 2-2".

Reply:
so nice. :+1:  thanks.